### PR TITLE
Experience Cloud Site login page Development from scratch using Aura

### PR DESCRIPTION
**Pull request by ManishKumarGupta8972**
Ticket included : [AVEBLOG__0053](https://theavenoir.atlassian.net/browse/AVEBLOG-70)
Issue : Experience Cloud Site login page Development from scratch using Aura
Cause : N/A
fix: N/A

### CODE REVIEW

File: force-app/main/default/aura/LoginForm/LoginFormHelper.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+({
+    handleLogin: function (component, event, helpler) {
```
Comment (developerAshish45) [inline force-app/main/default/aura/LoginForm/LoginFormHelper.js:2]: remove all var variable and use let

File: force-app/main/default/aura/LoginForm/LoginFormHelper.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                component.set("v.showError", true);
+                if (rtnValue === "success") {
+                    console.log("Login successful!");
+                }
+            } else if (state === "ERROR") {
+                var errors = response.getError();
+                if (errors) {
+                    console.error(errors);
+                }
+                component.set("v.errorMessage", "An unexpected error occurred. Please try again later.");
```
Comment (developerAshish45) [inline force-app/main/default/aura/LoginForm/LoginFormHelper.js:23]: use a custom label to show any message.

File: force-app/main/default/aura/LoginForm/LoginFormHelper.js (hunk 1 above)
Comment (ManishKumarGupta8972) [reply force-app/main/default/aura/LoginForm/LoginFormHelper.js:2]: Resolved

File: force-app/main/default/aura/LoginForm/LoginFormHelper.js (hunk 2 above)
Comment (ManishKumarGupta8972) [reply force-app/main/default/aura/LoginForm/LoginFormHelper.js]: Resolved

**Review by developerAshish45 — APPROVED** [review]

**Pull request merged**